src/settings/__convertToCssRemUnit.ts

```diff
@@ -0,0 +1,4 @@
+const convertFromPixelsToRem = (size: number): string => {
```

Comment:
Nice one, can we move this to the `misc` file, please?

Reply:
resolved.